catalog/dags/common/popularity/sql.py

```diff
@@ -191,7 +198,121 @@ def update_media_popularity_metrics(
         ;
         """
     )
-    postgres.run(query)
+    return postgres.run(query)
+
+
+@task
+def calculate_media_popularity_percentile_value(
+    postgres_conn_id,
+    provider,
+    media_type=IMAGE,
+    popularity_metrics_table=IMAGE_POPULARITY_METRICS_TABLE_NAME,
+    popularity_percentile=IMAGE_POPULARITY_PERCENTILE_FUNCTION,
+    task: AbstractOperator = None,
+):
+    if media_type == AUDIO:
+        popularity_metrics_table = AUDIO_POPULARITY_METRICS_TABLE_NAME
+        popularity_percentile = AUDIO_POPULARITY_PERCENTILE_FUNCTION
```

Comment:
This is an example of following the existing conventions in this file, that will be cleaned up in #2678. It's not easy to refactor it in this PR without also touching a lot of other methods, and I felt that would make this harder to review.